src/main/java/com/google/cloud/tools/gradle/appengine/core/AppEngineCorePluginConfiguration.java

```diff
@@ -79,6 +82,32 @@ public void execute(Project project) {
         });
   }
 
+  private void createManagedSdkTask() {
+    ManagedSdkTask managedSdkTask =
```

Comment:
I realize I wrote mangedSdkTask in the doc, but perhaps this should be downloadCloudSdk or something.